### ISSUE TITLE
Subroutine signatures require Perl 5.20+

### ISSUE THREAD

**Issue by eserte** [shlomif/app-notifier#1]
The test suite fsils with older perls (5.14 .. 5.18):
```
#   Failed test 'App/Notifier/Service.pm loaded ok'
#   at t/00-compile.t line 46.
#          got: '512'
#     expected: '0'
Subroutine signatures require Perl 5.20+ at /home/cpansand/.cpan/build/2021112422/App-Notifier-Service-0.0901-0/blib/lib/App/Notifier/Service.pm line 5.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2021112422/App-Notifier-Service-0.0901-0/blib/lib/App/Notifier/Service.pm line 5.
Compilation failed in require at -e line 1.
# Looks like you failed 1 test of 3.
t/00-compile.t ....... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
	(less 2 skipped subtests: 0 okay)
... etc ...
```

**Comment by shlomif** [comment]
Thanks! hopefully fixed in 0.0902.

**Comment by shlomif** [comment]
Closing as FIXED.
